libpod/container_validate.go

```diff
@@ -85,6 +86,15 @@ func (c *Container) validate() error {
 		return fmt.Errorf("cannot set static IP or MAC address if not creating a network namespace: %w", define.ErrInvalidArg)
 	}
 
+	// issue #25927 network-aliases cannot have spaces
+	for _, net := range c.config.Networks {
+		for _, alias := range net.Aliases {
+			if strings.Contains(alias, " ") {
+				return errors.New("invalid alias: contains white space")
```

Comment:
Include the alias in the error.
				return errors.New("invalid alias %q: contains white space", alias)